builtin/game/item.lua

```diff
@@ -644,7 +638,6 @@ core.noneitemdef_default = {  -- This is used for the hand and unknown items
 
 	-- Interaction callbacks
 	on_place = redef_wrapper(core, 'item_place'),
-	on_secondary_use = redef_wrapper(core, 'item_secondary_use'),
```

Comment:
Yea, not even slower, because tables don't contain `nil` values. The compiler might just ignore it.
I've moved it behind on_use.